src/screens/Onboarding/steps/useCaseSelection.js

```diff
@@ -1,17 +1,14 @@
 // @flow
```

Comment:
not for QA to test this page in the onboarding. Settings > Set up New Device > Nano S/X -> This page